be/src/storage/tablet_updates.cpp

```diff
@@ -1133,6 +1142,15 @@ Status TabletUpdates::_commit_compaction(std::unique_ptr<CompactionInfo>* pinfo,
     edit_version_info->rowsets.swap(nrs);
     edit_version_info->compaction.swap(*pinfo);
     _edit_version_infos.emplace_back(std::move(edit_version_info));
+    if (_edit_version_infos.size() >= 2) {
```

Comment:
same logic, how about abstract out this code snippet into a function like "check_if_creation_time_valid"?

Reply:
The downside of this is LOG will be at the same line, losing information about which function checks this

Reply:
indeed
i mean the check procedure, not the log line
for example:
if (check_if_creation_time_valid()) {
   // log line..
}
but never mind, this implement also ok.